roles/network_plugin/calico/templates/calico-cr.yml.j2

```diff
@@ -19,11 +19,16 @@ rules:
     verbs:
       - watch
       - list
+{% if calico_datastore == "kdd" %}
```

Comment:
This will create newlines and inline comments. I don't think that is supported

Reply:
Hello @woopstar. I have made checks for calico_datastore="etcd" and calico_datastore="kdd" and calico-cr.yml doesn't contain newlines and inline comments.

My ansible version is 2.7.7.